nginx-controller/nginx/templates/nginx.tmpl

```diff
@@ -75,5 +75,6 @@ http {
         }
     }
 
+    include /etc/nginx/maps*.conf;
```

Comment:
`/etc/nginx/maps*.conf` not sure why we have `*`

Reply:
Because maps.conf can be not created at the moment nginx started for the first time.

Reply:
@seletskiy it is possible to create it [here](https://github.com/nginxinc/kubernetes-ingress/blob/master/nginx-controller/nginx/nginx.go#L148) along with the main config.

Reply:
+